account_invoice_overdue_reminder/models/account_move.py

```diff
@@ -61,7 +61,7 @@ def _compute_overdue_reminder(self):
             all_dates = []
             all_counters = []
             # faster than 2 list comprehension because we loop only once?
-            for reminder in move.overdue_reminder_ids:
+            for reminder in move.overdue_reminder_ids.filtered(lambda m: m.action_date):
```

Comment:
This can be put in a shortest way:

```suggestion
            for reminder in move.overdue_reminder_ids.filtered("action_date"):
```